src/main/java/ro/msg/learning/shop/exception/LocationOutOfStockException.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.exception;
+
+import lombok.Getter;
+
+import java.util.UUID;
+
+@Getter
+public class LocationOutOfStockException extends RuntimeException {
+    private final UUID locationId;
```

Comment:
Do you really need these variables? Do you use them elsewhere than the constructor in order to set the error message?

Reply:
Removed them